hibernate/cache-commons/src/test/java/org/infinispan/test/hibernate/cache/commons/tm/NarayanaStandaloneJtaPlatform.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.infinispan.test.hibernate.cache.commons.tm;
+
+
+import javax.transaction.TransactionManager;
+import javax.transaction.UserTransaction;
+
+import org.hibernate.engine.transaction.jta.platform.internal.AbstractJtaPlatform;
+import org.hibernate.engine.transaction.jta.platform.spi.JtaPlatformException;
+
+public class NarayanaStandaloneJtaPlatform extends AbstractJtaPlatform {
```

Comment:
Is my Intellij playing tricks on me or is this file really using tabs instead of spaces? We have a checkstyle rule against that, but it was (temporarily) disabled for some modules.

Reply:
sorry, copy pasted this from Search.